decidim-budgets/lib/decidim/budgets/project_serializer.rb

```diff
@@ -14,7 +14,8 @@ def initialize(project)
 
       # Public: Exports a hash with the serialized data for this project.
       def serialize
-        {
+        finalize(
```

Comment:
Can we apply this to all serializers, so we could hook into all of them?

Reply:
Added finalize to all serializers